src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -98,14 +92,15 @@ const useGroupInfo = () => {
       const sendData = {
         user_id: selectedUser.user_id
       };
+      const members = [];
+      members.push(profile?.myProfile?.user_id, selectedUser?.user_id);
       const processGetBlock = await checkUserBlock(sendData);
-
       if (!processGetBlock.data.data.blocked && !processGetBlock.data.data.blocker) {
-        return openChatMessage();
+        return createSignChat(members, selectedUser);
       }
       return handleOpenProfile(selectedUser);
     } catch (e) {
-      console.log(e, 'eman');
+      console.log('error:', e);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for checking user block status and creating a signed chat has been updated. The error handling has also been improved with more specific error messages. However, please ensure that the new functions `createSignChat` and `handleOpenProfile` are implemented correctly and handle all edge cases.

```diff
- return openChatMessage();
+ return createSignChat(members, selectedUser);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->